extra/1-radio-stations.js

```diff
@@ -14,10 +14,18 @@
  */
 
 // `getAllFrequencies` goes here
-
+function getAllFrequencies(){
+  let frequencies = [];
+  for (let i = 87; i <= 108;i++){
+  frequencie.push(i);
```

Comment:
You're refrencing frequencie instead of frequencies, this will error out as frequencie  is not defined